WordPress/WordPressShareExtension/MainShareViewController.swift

```diff
@@ -96,13 +115,9 @@ private extension MainShareViewController {
 
         let shareNavController = UINavigationController(rootViewController: editorController)
 
-        if #available(iOS 13, *), editorController.originatingExtension == .saveToDraft {
-            // iOS 13 has proper animations and presentations for share and action sheets. So the `else` case should be removed when iOS 13 is minimum.
+        if editorController.originatingExtension == .saveToDraft {
             // We  need to make sure we don't end up with stacked modal view controllers by using this:
             shareNavController.modalPresentationStyle = .overFullScreen
-        } else {
-            shareNavController.transitioningDelegate = extensionTransitioningManager
```

Comment:
`ExtensionTransitioningManager` is now unused. I didn't remove it in the context of this PR because removing it triggers a cascade of removals which felt more appropriate for a dedicated PR.